### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JamesMBallard**

### CODE REVIEW

File: calcite/README.md [hunk 1]
```diff
@@ -107,4 +107,4 @@ For further details see [Calcite Web TypeFaces](https://esri.github.io/calcite-w
 
 ## Demo app
 
-In the [demo](./demo/) folder, there is a Qt demo application project (calcite_test.pro) that showcases many of the Qt toolkit Calcite components being used.
\ No newline at end of file
+In the [demo](./demo/) folder, there is a Qt demo application project (calcite_test.pro) that showcases many of the Qt toolkit Calcite components being used.
```
Comment (JamesMBallard) [inline calcite/README.md:110]: I am not sure why this change shows up. I didn't change anything here. I only changed the typo above.

**Review by jared-2016 — APPROVED** [review]
